dev/js/ysFilter.js

```diff
@@ -87,12 +97,24 @@
 				paramTypes = $this.filter.settings.filter;
 				totalItems = $this.filter.filter;
 
+				var categoryRegex = '^' + $this.set.category;
```

Comment:
This should be the default and only written over further down. This should however be for exact matching of the categories.